notebooks/official/automl/automl_tabular_on_vertex_pipelines.ipynb

```diff
@@ -0,0 +1,1083 @@
+{
```

Comment:
<p>Please remove "Note: Calculation of model evaluation is not supported for VPC-SC environment. The support will come soon.". Now it's supported. I have updated the colab accordingly.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/694/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>